examples/compiled/bar_multi_values_per_categories.vg.json

```diff
@@ -51,20 +51,65 @@
   ],
```

Comment:
The SVG changed here in odd ways. There is now a while line around the bar.

Reply:
Actually there was a white line around the bar before, but now it's cut into half.

Reply:
Because the mask clips everything, which include the part of the stroke outside.
This'll be a difficult one. I'm almost thinking we should apply the mask only when cornerRadius is used. Any thoughts?

Reply:
Also, it's a bit weird that stroke applies to the individual bars, but cornerRadius applies to the entire stack.

Reply:
It's somewhat weird but probably what people would expect. Also, you cannot easily set the stroke on the whole bar but also color it for each segment.

If this becomes an issue in the future, we could add a property that says whether stacked bars should be in a single group or not.

Reply:
I'm trying to add the stroke to group, still having some issues (see this image), so let's not merge this one yet.

Reply:
Vega is adding a 0.5px offset to group strokes (probably to align the plot areas to pixel boundaries). I opened an issue here: https://github.com/vega/vega/issues/2186.

Reply:
You fixed this, right?

Reply:
Yes, I'm using a workaround --- add another layer for the stroke if specified.

Reply:
We can now remove the workaround, right?

Reply:
We have to wait until Vega fixes https://github.com/vega/vega/issues/2186 and https://github.com/vega/vega/issues/2197. Thank you for opening https://github.com/vega/vega/issues/2197, that's also necessary.

Reply:
I'm looking into fixing a bunch of config typings right now. Could you try to send a pull request to fix https://github.com/vega/vega/issues/2197?

Reply:
Done! Let's make this WIP until Vega fixes the issue.